openstack_hypervisor/pci.py

```diff
@@ -106,6 +106,15 @@ def is_network_device(pci_class: str) -> bool:
     return pci_class.startswith("0x02")
 
 
+def is_display_device(pci_class: str) -> bool:
+    """Specifies whether the PCI class represents a display device."""
+    # PCI class format:
+    #   * class code (8 bytes)
+    #   * subclass code (8 bytes)
+    #   * vendor specific (8 bytes)
+    return pci_class.startswith("0x03")
```

Comment:
how did you come up with this value?, should we make this a global constant to make it more readable?

Reply:
From https://admin.pci-ids.ucw.cz/read/PD. Modified as global constant